data/settings/src/main/proto/com/google/jetpackcamera/settings/target_frame_rate.proto

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+option java_package = "com.google.jetpackcamera.settings";
+option java_multiple_files = true;
+
+enum TargetFrameRate {
+  TARGET_FRAME_RATE_UNDEFINED = 0;
+  TARGET_FRAME_RATE_NONE = 1;
```

Comment:
Why do we need UNDEFINED and NONE? What is the difference?

Reply:
so `NONE` is the actual user/app defined choice to not use a target frame rate.
`UNDEFINED` is there to follow proto best practices for enums, to leave a default unspecified value to ensure consistent expected behavior when adding new values.
https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum
>Enums should include a default `FOO_UNSPECIFIED` value as the first value in the declaration . When new values are added to a proto2 enum, old clients will see the field as unset and the getter will return the default value or the first-declared value if no default exists 

I could add a comment to the enum files with that explanation, and maybe rename undefined to remove the ambiguity between none?

Reply:
following this [recommended practice](https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum) for proto enums by setting an unspecified value for the first enum ordinal. 
>When new values are added to a proto2 enum, old clients will see the field as unset and the getter will return the default value or the first-declared value if no default exists . For consistent behavior with [proto enums](https://protobuf.dev/programming-guides/proto2/#enum), the first declared enum value should be a default FOO_UNSPECIFIED value and should use tag 0.

I _could_ rename it or add a comment to address the ambiguity, but based on [this comment](https://github.com/google/jetpack-camera-app/pull/86#discussion_r1518444032) I'll likely scrap that enum anyways.

Reply:
Thanks for the context. I'll make sure to add UNDEFINED for all my proto enums.

But yes, I don't think this is needed; we could just use integers. So in that case you can remove this.